From c3b9fbbb7168003bdf09fa4e35652bbe308330d8 Mon Sep 17 00:00:00 2001
From: Johannes Kastl <kastl@b1-systems.de>
Date: Fri, 16 Nov 2018 21:14:46 +0100
Subject: [PATCH] do not abuse relative paths in docker plugin to make docker
 work, added by tmkn@tmkn.uk on Thu Oct 26 19:42:46 UTC 2017

Signed-off-by: Johannes Kastl <kastl@b1-systems.de>
---
 plugins/providers/docker/config.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/providers/docker/config.rb b/plugins/providers/docker/config.rb
index b3b6993c6..f9b147597 100644
--- a/plugins/providers/docker/config.rb
+++ b/plugins/providers/docker/config.rb
@@ -1,6 +1,6 @@
 require "pathname"
 
-require_relative "../../../lib/vagrant/util/platform"
+require "vagrant/util/platform"
 
 module VagrantPlugins
   module DockerProvider
-- 
2.19.1

